flink-container/src/main/java/org/apache/flink/container/entrypoint/StandaloneJobClusterEntryPoint.java

```diff
@@ -53,26 +53,31 @@
 	private final String[] programArguments;
 
 	@Nullable
-	private final String jobClassName;
+	private final String jobEntryPointName;
+
+	@Nullable
+	private final String jobPythonArtifacts;
```

Comment:
I think it is not a good idea to add special case logic into the `StandaloneJobClusterEntrypoint`. Why do you think this is needed?